libs/pubsub: Benchmark*ClientsOneQuery fails with "already subscribed"

<!--
Please fill in as much of the template below as you can.

Be ready for followup questions, and please respond in a timely
manner. We might ask you to provide additional logs and data (tendermint & app).
-->

**Tendermint version** (use `tendermint version` or `git rev-parse --verify HEAD` if installed from source):
Not Applicable


**What happened**:
I tried to run benchmarks using Bencher

**What you expected to happen**:
Successfully ran benchmarks

**Have you tried the latest version**: yes/no
Yes

**How to reproduce it** (as minimally and precisely as possible):
Just try to run `go test -run=^$ -bench=.`

**Logs (paste a small part showing an error (< 10 lines) or link a pastebin, gist, etc. containing more of the log file)**:
```shell
 go test -run=^$ -bench=.
goos: darwin
goarch: amd64
pkg: github.com/tendermint/tendermint/libs/pubsub
cpu: Intel(R) Core(TM) i7-7920HQ CPU @ 3.10GHz
Benchmark10Clients-8             	  195499	      6078 ns/op	    1920 B/op	      28 allocs/op
Benchmark100Clients-8            	  175038	      6138 ns/op	    1909 B/op	      28 allocs/op
Benchmark1000Clients-8           	  168866	      6687 ns/op	    1960 B/op	      29 allocs/op
--- FAIL: Benchmark10ClientsOneQuery
    pubsub_test.go:523: already subscribed
--- FAIL: Benchmark100ClientsOneQuery
    pubsub_test.go:523: already subscribed
--- FAIL: Benchmark1000ClientsOneQuery
    pubsub_test.go:523: already subscribed
FAIL
exit status 1
FAIL	github.com/tendermint/tendermint/libs/pubsub	5.214s
```

**Anything else we need to know**:
Bencher will help continuously run benchmarks and avoid such problems early enough. Please check out https://bencher.orijtech.com/

--- a/libs/pubsub/pubsub_test.go
+++ b/libs/pubsub/pubsub_test.go
@@ -518,7 +518,8 @@ func benchmarkNClientsOneQuery(n int, b *testing.B) {
 	ctx := context.Background()
 	q := query.MustParse("abci.Account.Owner = 'Ivan' AND abci.Invoices.Number = 1")
 	for i := 0; i < n; i++ {
-		subscription, err := s.Subscribe(ctx, clientID, q)
+		id := fmt.Sprintf("clientID-%d", i+1)
+		subscription, err := s.Subscribe(ctx, id, q)
 		if err != nil {
 			b.Fatal(err)
 		}

